tests/components/homekit_controller/test_config_flow.py

```diff
@@ -60,13 +74,21 @@
 ]
 
 
-def _setup_flow_handler(hass):
+def _setup_flow_handler(hass, pairing=None):
     flow = config_flow.HomekitControllerFlowHandler()
```

Comment:
In another PR this should preferably be made more robust by using the core interfaces to set up the tests. Ie start the config flow with `hass.config_entries.flow.async_init`. See the hue config flow tests for good examples.

Reply:
Yeah i think i saw your PR doing this to an integration a little while back and put it on my todo list (see PR description). Thanks for the pointer to a good example.